### ISSUE TITLE
:art: Créer epository's social media preview (640 by 320 pixels.)

### ISSUE THREAD

**Issue by adriens** [lschaeffer313/domaine-nc-mobile#134]
cf [doc](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview)

**Comment by adriens** [comment]
Vous en dites quoi @lschaeffer313  @HakumenNC :grey_question: 
![Designer](https://user-images.githubusercontent.com/5235127/215005659-10cb5650-30b6-4870-96ef-c5c63bcb1911.png)

**Comment by adriens** [comment]
AH.. mieux : 

![Designer(1)](https://user-images.githubusercontent.com/5235127/215005994-25d4ee8f-73b8-472b-a2ca-9198fc839516.png)

**Comment by adriens** [comment]
A la palce de Flutter, ptet un logo bien clean de l'appl cf 

- https://github.com/lschaeffer313/domaine-nc-mobile/issues/113
- https://github.com/lschaeffer313/domaine-nc-mobile/issues/133

**Comment by lschaeffer313** [comment]
J'aime bien. Il manque que le logo et c'est nickel.

**Comment by adriens** [comment]
> J'aime bien. Il manque que le logo et c'est nickel.

Oui, on le met en normal ? Droppe moi celui que tu veux que je charge et je te fais ça fissa :zap:

**Comment by lschaeffer313** [comment]
Je pense m'en occuper dans le week-end

**Comment by lschaeffer313** [comment]
Autant le faire créer par une IA 🤣
![image](https://user-images.githubusercontent.com/28922072/215025478-fbe1183a-98c9-4e3b-9953-71dce6d3ae78.jpeg)

Lequel préfère-tu ?

**Comment by HakumenNC** [comment]
Stylé, ne pas oublier la version "themed" cf. #113, ce logo peut s'y preter 

![image](https://user-images.githubusercontent.com/2644417/215028393-87e92a5e-abe2-4fd0-aed5-308b23903e02.png)

**Comment by HakumenNC** [comment]
> AH.. mieux :
> 
> ![Designer(1)](https://user-images.githubusercontent.com/5235127/215005994-25d4ee8f-73b8-472b-a2ca-9198fc839516.png)

Trouver un logo Flutter avec un fond transparent, ça se tente, si ça rend bien?

**Comment by adriens** [comment]
@lschaeffer313  : ton idée déchire ton C TOTALEMENT DISRUPTIF :heavy_heart_exclamation:  :star_struck: :heart_eyes: 
@HakumenNC  : je sais pas si on laisse la stack en fait, c'était surtout pour monter un logo

:thinking: Avec les nouveaux designs produits par @lschaeffer313 , je vais ptet regénérer mon draft totalement !

**Comment by adriens** [comment]
> Stylé, ne pas oublier la version "themed" cf. https://github.com/lschaeffer313/domaine-nc-mobile/issues/113, ce logo peut s'y preter

Yep, tout à fait, avec un design bien à plat limite vectoriel pour qu'il soit super propre sur le bureau....

**Comment by adriens** [comment]
> Lequel préfère-tu ?
> PS: J'adore Midjourney heart

Lance un poll sur les discussions :rocket:

**Comment by adriens** [comment]
@lschaeffer313 : droppe tes prompts :pray:

**Comment by lschaeffer313** [comment]
Ah mince j'ai déjà fait l'upscale sur le troisième 😅

**Comment by lschaeffer313** [comment]
Je check si je peux pas faire retour en arrière

**Comment by adriens** [comment]
Et le `.nc` :grey_question:

**Comment by adriens** [comment]
:trollface:

**Comment by adriens** [comment]
![image](https://user-images.githubusercontent.com/5235127/215034740-53549a7d-8bd7-41a0-96ca-6c5e6e96b017.png)

**Comment by adriens** [comment]
![image](https://user-images.githubusercontent.com/5235127/215034965-c0517da1-df8a-4f24-ae85-81e66536b3e6.png)

**Comment by adriens** [comment]
https://discord.com/channels/662267976984297473/1008571111530836068/1068438909035499530

![image](https://user-images.githubusercontent.com/5235127/215036038-7a51ba22-bb77-44a5-a8b4-60e20d9ab816.png)

**Comment by adriens** [comment]
@lschaeffer313 @HakumenNC : https://github.com/invoke-ai/InvokeAI

**Comment by adriens** [comment]
Désolé les gars, j'ai pas gardé le prompt :sweat_smile:
